NerdyNomads/Text-Savvy
# Re/feat/extension workspace

## What does this PR do?
Chrome context menu displays workspace from database. Highlight and save text is saved on to the workspace **texts** array and the workspace ID is also attached to the saved text.

### Screenshots (optional)
![image](https://user-images.githubusercontent.com/72952442/158299073-960e54d5-f474-43e6-a676-670e95038054.png)

### Steps to test/review (optional)
- start back-end (front-end optional)
- build the extension and load into chrome browser
- highlight and save while checking the database to see changes: 
    - workspace ID where text was saved is included
    - new text is pushed on to the workspace it was saved in

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues

#96
